### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by FranzGB**
- Add logo to assets folder
- Add fonts to assets folder
- Add css classes files for navbar

### CODE REVIEW

File: css/layout/layout.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
+.header__main {
```
Comment (well1791) [inline css/layout/layout.css:1]: the layout mindset is good, but, a header is more of a component than a layout, but (again) on the other hand, if you can place content on top of your layout class then you good to go with a layout, a good example of this is css grid.

File: css/layout/layout.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  margin: auto;
+  margin-top: 1.25rem;
+  width: 100%;
+  max-width: 332px;
+  padding: 1.25rem 1.25rem;
```
Comment (well1791) [inline css/layout/layout.css]: if the size is the same you could use just `padding: 1.25rem`

File: css/layout/layout.css [hunk 3]
```diff
@@ -0,0 +1,25 @@
+.header__main {
+  margin: auto;
+  margin-top: 1.25rem;
```
Comment (well1791) [inline css/layout/layout.css:3]: good job

File: css/components/navbar.css [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+.nav__item {
+  list-style: none;
+}
+
+.nav__link {
+  text-decoration: none;
+  color: #000;
+  font-weight: 400;
+  padding: 0.75rem 1.25rem;
```
Comment (well1791) [inline css/components/navbar.css]: you will need a `transition` to make this smooth

File: css/components/buttons.css [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+  cursor: pointer;
+}
+
+.button__secondary:hover {
+  background-color: #FFF;
+}
+
+.button__secondary:focus {
+  background-color: #FFF;
+}
```
Comment (well1791) [inline css/components/buttons.css]: you could have done this the same way you did with `:focus, :focus-within`, mind to refactor this properly

File: css/components/navbar.css [hunk 6]
```diff
@@ -0,0 +1,100 @@
+.nav__bar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 3.25rem;
```
Comment (well1791) [inline css/components/navbar.css]: instead of using height you could let your inside elements define it for you

File: css/components/navbar.css [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+  width: auto;
+}
+
+.nav__toggle__button span {
+  width: 3.25rem;
+  font-size: 2rem;
+  text-align: center;
+  padding: 0;
+  cursor: pointer;
```
Comment (well1791) [inline css/components/navbar.css]: not good to add `pointer` on a `span`, instead the container element `button` should have it, and the children will inherit it

File: css/components/navbar.css [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+.nav__logo__img {
+  height: 3.25rem;
+  width: auto;
+}
+
+.nav__toggle__button span {
+  width: 3.25rem;
+  font-size: 2rem;
+  text-align: center;
+  padding: 0;
```
Comment (well1791) [inline css/components/navbar.css]: if you need to set padding as 0 on a span, then, definitely there is something wrong

File: css/components/navbar.css [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+  padding: 1rem 1.25rem;
+  align-self: start;
+
+  .nav__list {
+    flex-direction: column;
+  }
+
+  .nav__button__group {
+    flex-direction: column;
+  }
```
Comment (well1791) [inline css/components/navbar.css]: remember this is plain css, not sure if the container feature is working by default, but for now, let's not take advantage of it, these lines should be written as `.nav__toggle__button:focus+.nav__menu, .nav__toggle__button:focus-within+.nav__menu .nav__list {` and `.nav__toggle__button:focus+.nav__menu, .nav__toggle__button:focus-within+.nav__menu .nav__button__group {`

Comment (FranzGB) [reply css/components/navbar.css]: > remember this is plain css, not sure if the container feature is working by default, but for now, let's not take advantage of it, these lines should be written as `.nav__toggle__button:focus+.nav__menu, .nav__toggle__button:focus-within+.nav__menu .nav__list {` and `.nav__toggle__button:focus+.nav__menu, .nav__toggle__button:focus-within+.nav__menu .nav__button__group {`

This works by default but I can separate this thanks for the feedback 🙇

**Pull request merged**